macros/minversionGeneric.ejs

```diff
@@ -76,18 +76,18 @@ switch(lang) {
         break;
     default: break;
 }
- 
+
 switch($0) {
     case 'inline':
-        %><span class="inlineIndicator standardNote standardNoteInline"><%=newIn%><%- web.link(wiki.uri(link), productVersion) %></span><%
+        %><span class="inlineIndicator standardNote"><%=newIn%><%- web.link(wiki.uri(link), productVersion) %></span><%
         break;
     case 'header':
-        %><div class="overheadIndicator standardNote standardNoteBlock">
+        %><div class="blockIndicator standardNote">
 <p><%-text%></p>
 </div><%
         break;
     case 'note':
-        %><div class="blockIndicator standardNote standardNoteBlock">
+        %><div class="blockIndicator standardNote">
```

Comment:
Did not find any pages that uses this macro:
https://developer.mozilla.org/en-US/search?topic=none&kumascript_macros=minversionGeneric&locale=*

Reply:
This is used in many minversion type macros.